uw-frame-components/portal/notifications/directives.js

```diff
@@ -11,16 +11,17 @@ define(['angular', 'require'], function(angular, require){
         }
     });
 
-    app.directive('notificationBell', function(){
+    app.directive('notificationPartials', function(){
```

Comment:
Good point that it's `frame` rather than `angularJS-portal`. I had latched onto the notification-ness of it, which is partly about making the frame-based app feel like it's part of the portal, but would need a namespace that makes sense more widely within frame apps.

`uwf-` has some nice properties in matching the name of the larger software product and *not* grabbing the whole `uw-` namespace.